README.md

```diff
@@ -42,22 +52,50 @@ or on any other Ubuntu (>= 22.04):
 - Clone the current repo
 - Generate TD guest image
 
-  The original image is ubuntu-23.10-server-cloudimg-amd64.img and it can be foud here
-	https://cloud-images.ubuntu.com/releases/mantic/release/
-
+```bash
   $ (cd  guest-tools/image/ ; sudo ./create-td-image.sh)
+```
+
+This script will create a TD image based on the Ubuntu 23.10 Cloud Image ubuntu-23.10-server-cloudimg-amd64.img
+and can be found at : https://cloud-images.ubuntu.com/releases/mantic/release/
+
+This base image can be customized by setting two environment variables:
+
+```bash
+  $ export OFFICIAL_UBUNTU_IMAGE="https://cloud-images.ubuntu.com/releases/mantic/release/"
+  $ export CLOUD_IMG="ubuntu-23.10-server-cloudimg-amd64.img"
+```
 
 The TD image will be available at : /tmp/tdx-guest-ubuntu-23.10.qcow2
-The root passwordis set to 123456.
+
+The root password is set to 123456
 
 ## Run a TD guest with QEMU
 
 On the TDX host previously setup, to run the TD guest with QEMU:
 
+If you have cloned this repository, the script wil be in guest-tools.
+
+```bash
+  $ cd guest-tools
+```
+
+If not, you can download the script as follow:
+
+```bash
+  $ https://raw.githubusercontent.com/canonical/tdx/dev-review/guest-tools/run_td.sh
+  $ chmod a+x run_td.sh
+```
+
+To run the TD:
+
+```bash
   $ TD_IMG=/tmp/tdx-guest-ubuntu-23.10.qcow2 ./run_td.sh
```

Comment:
Do you think we can remind the users to move the created qcow2 from /tmp to a common folder? I'm not sure, this is not a TDX specific thing, the files in /tmp will be removed after reboot, just in case the users don't remember.(I forgot it myself in the beginning...)

Reply:
Ok i will do that, thanks